lua/downloader/cache.lua

```diff
@@ -0,0 +1,23 @@
+local MODULE = {}
+MODULE.Priority = 3
+
+function MODULE:Run(context)
+    -- cache = { [number wsid] = number updated, ... }
+    local cacheFile = context.dataFolder .. "/workshop_cache.txt"
+    local cache = file.Exists(cacheFile, "DATA") and util.JSONToTable(file.Read(cacheFile, "DATA")) or {}
```

Comment:
no need to check if file exists:
```suggestion
    local cache = util.JSONToTable(file.Read(cacheFile, "DATA") or "{}")
```

Reply:
Ok.